testing/coreruleset_test.go

```diff
@@ -2,8 +2,6 @@
 // SPDX-License-Identifier: Apache-2.0
 
 // These benchmarks don't currently compile with TinyGo
-//go:build !tinygo
```

Comment:
🤓 @anuraaga

Reply:
Oh nice! Can you remove the above comment too?

Reply:
:facepalm:

Reply:
Somehow they don't work on CI and nor in my computer anymore 🤦